Dockerfile

```diff
@@ -36,15 +36,21 @@ WORKDIR /build/pufferpanel
 COPY go.mod go.sum ./
 RUN go mod download && go mod verify
 
+COPY client/package.json client/package-lock.json ./client/
```

Comment:
This causes issues if you work on Windows + Linux. Copy will pull from the source, and so the COPY later will stomp on any changes this does.